server/sonar-web/src/main/js/apps/issues/conciseIssuesList/ConciseIssueLocations.js

```diff
@@ -20,25 +20,29 @@
 // @flow
 import React from 'react';
 import ConciseIssueLocationBadge from './ConciseIssueLocationBadge';
-import type { Issue } from '../../../components/issue/types';
+/*:: import type { Issue } from '../../../components/issue/types'; */
 
+/*::
 type Props = {|
   issue: Issue,
   onFlowSelect: number => void,
   selectedFlowIndex: ?number
 |};
+*/
 
+/*::
 type State = {
   collapsed: boolean
 };
+*/
 
 const LIMIT = 3;
 
 export default class ConciseIssueLocations extends React.PureComponent {
-  props: Props;
-  state: State = { collapsed: true };
+  /*:: props: Props; */
+  state /*: State */ = { collapsed: true };
 
-  handleExpandClick = (event: Event) => {
+  handleExpandClick = (event /*: Event */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)